cmd/operator/main.go

```diff
@@ -119,19 +117,10 @@ func main() {
 }
 
 func runCLI(logger *logrus.Logger, cfg *config.Config) {
-
-	runclient, err := run.NewAPIClient(context.Background(), flRegions[0])
-	if err != nil {
-		logger.Fatalf("could not initilize Cloud Run client: %v", err)
-	}
-	roll := rollout.New(&service.Client{
-		RunClient:   runclient,
-		Project:     cfg.Targets[0].Project,
-		ServiceName: "hello",
-		Region:      flRegions[0],
-	}, cfg.Strategy).WithLogger(logger)
-
 	for {
+		services := service.Filter(cfg)
+		roll := rollout.New(services[0], cfg.Strategy).WithLogger(logger)
+
 		changed, err := roll.Rollout()
 		if err != nil {
 			logger.Infof("Rollout failed: %v", err)
```

Comment:
re: this code block (you can address in a different PR)

```
		interval := time.Duration(cfg.Strategy.Interval)
		time.Sleep(interval * time.Second)

```

we need to find a better name for "Interval". (e.g. "What interval is this for?")

Reply:
Yes, we should probably make it clearer. We can change it in another PR since this interval is only used here.